packages/bootleg-tokens/contracts/ISharedRoyaltyTokenEnumerable.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.3;
+
+/**
+ * @title Shared Royalty NFT Enumerable interface for franchisors
+ */
+contract ISharedRoyaltyTokenEnumerable {
+
+    function totalSupply() public view returns (uint256);

```

Comment:
I was thinking that anything in the [ERC721Enumerable](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/token/ERC721/IERC721Enumerable.sol) we could just pull from there rather than copying here. 

In reference to `totalSupply`, `tokenOfOwnerByIndex`, and `tokenByIndex`.

It could make sense to make this one inherit from the other. Not sure I have a well-reasoned opinion for making devs down the road wanting them separately (not inheriting, more typing) vs always wanting everything enumerable if it's going to be enumerable (inheriting, less typing).

Reply:
Having this in here was a mistake. I copied it there but forgot to remove it before I pushed my code. I've removed these.